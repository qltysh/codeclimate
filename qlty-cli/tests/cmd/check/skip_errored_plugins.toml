bin.name = "qlty"
args = ["check", "--all", "--no-cache", "--skip-errored-plugins"]
env.remove = ["RUST_BACKTRACE"]