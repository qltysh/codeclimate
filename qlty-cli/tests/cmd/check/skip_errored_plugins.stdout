Lint error always_error: Exited with code 1 .qlty/out/invoke-[..].yaml

